### PR TITLE
[T4A3][T11-B3] Teh Chee Yeo

### PR DESCRIPTION
**Pull request by muahcheeee**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
+ */
+public class Contact {
+	
+    public String value;
```
Comment (kychua) [inline src/seedu/addressbook/data/person/Contact.java:8]: why is this public?

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contact in the address book.
```
Comment (kychua) [inline src/seedu/addressbook/data/person/Contact.java:4]: no need to mention where the class is used (i.e. in the address book)

**Comment by kychua** [comment]
@cheo1994 Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Contact.java (hunk 1 above)
Comment (muahcheeee) [reply src/seedu/addressbook/data/person/Contact.java:8]: I assumed the creator would like it to be public as seen in the children class thats why I left it as public.

**Pull request closed without merging**